pages/_document.tsx

```diff
@@ -29,6 +29,19 @@ export default class MyDocument extends Document {
             integrity='sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf'
             crossOrigin='anonymous'
           />
+          {/* light/dark theme favicons */}
+          {/* <link
+            rel='icon'
+            media='(prefers-color-scheme:dark)'
+            href='favicon-dark.png'
+            type='image/png'
+          />
+          <link
+            rel='icon'
+            media='(prefers-color-scheme:light)'
+            href='favicon-light.png'
+            type='image/png'
+          /> */}
```

Comment:
Why is this commented out?

Reply:
Jerry trying to sneak in that light & dark theme. I don't think it needs it, the entire website is black and white, just invert your screen colours and you done.

Reply:
Commented out coz we don't have favicon yet